config/README.md

```diff
@@ -123,13 +125,15 @@ your local machine using the kubectl command:
 
 ### Create a user account
 
-    kubectl -n $NAMESPACE exec cockroachdb-0 -ti -- \
+Pick a username and create an account:
+
+    kubectl -n dss-main exec cockroachdb-0 -ti -- \
         ./cockroach --certs-dir ./cockroach-certs \
         user set $USERNAME --password
 
 ### Access the web UI
 
-    kubectl -n $NAMESPACE port-forward cockroachdb-0 8080
+    kubectl -n dss-main port-forward cockroachdb-0 8080
```

Comment:
All DSS operators will have at least two DSS instances (prod and test environment) per DSS0210; shouldn't we keep $NAMESPACE to facilitate this?  If not, how would the two different DSS instances be addressed without $NAMESPACE?

Reply:
that's a bad practice since kubernetes namespaces are not fully isolated. They should be completely separate clusters

Reply:
Ok.  What was the original purpose of $NAMESPACE?  Are we just fixing the prior bad practice?

Reply:
it was an artifact of the original cockroachdb docs, that put the database in a separate namespace. These systems are so interrelated I don't think there's benefit in putting each in its own namespace.